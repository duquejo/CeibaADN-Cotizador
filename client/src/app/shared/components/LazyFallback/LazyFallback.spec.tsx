import * as React from 'react';
import { ShallowWrapper, mount, render, shallow } from 'enzyme';
import { LazyFallback } from './index';

describe('Pruebas sobre componente <LazyFallback />', () => {

    let componentWrapper: ShallowWrapper;

    beforeEach( () => {
        componentWrapper = shallow(<LazyFallback/>);
    });

    it('Debería renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });

    it('Debería tener el span personalizado implementado', () => {
        expect( componentWrapper.find( 'SpanFallback' ).length ).toBe(1);
    });
});