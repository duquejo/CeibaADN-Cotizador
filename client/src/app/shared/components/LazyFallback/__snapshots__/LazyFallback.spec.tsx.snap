// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <LazyFallback /> Debería renderizar el elemento 1`] = `
<styled.div>
  <SpanFallback>
    Cargando página...
  </SpanFallback>
</styled.div>
`;
